doc/ansible/dev/advanced_options.md

```diff
@@ -35,3 +35,11 @@ Some features can be overridden per resource via an annotation on that CR. The o
 ### Runner Directory
 
 The ansible runner will keep information about the ansible run in the container.  This is located `/tmp/ansible-operator/runner/<group>/<version>/<kind>/<namespace>/<name>`. To learn more  about the runner directory you can read the [ansible-runner docs](https://ansible-runner.readthedocs.io/en/latest/index.html).
+
+## Turning off Dependent Watches and Owner Reference Injection
+
+As user may want to manage what they watch and the owner references themseleves. This means that your operator will need to understand how to clean up after itslef when your CR is deleted. To disable these features you will need to edit your `build/Dockerfile` to include the line below.
```

Comment:
Maybe replace the first sentence with something more like `You may want to manage what your operator watches and the owner references themselves.`

`s/itslef/itself/`

`s/line below./line below:`

Reply:
Also I would say more explicitly that you cannot turn off ownerReferences without disabling dependent watches. Also perhaps describe what would happen if your watches file set `watchDependentResources: true` and inject owner references was disabled  (is it an error? Does the dependent watch just silently not happen?)